frontend/app/routes/$lang/_public/status/index.tsx

```diff
@@ -143,15 +143,17 @@ export default function StatusChecker() {
   const microsoftServiceAgreement = <InlineLink to={t('status:links.microsoft-service-agreement')} className="external-link" newTabIndicator target="_blank" />;
   const fileacomplaint = <InlineLink to={t('status:links.file-complaint')} className="external-link" newTabIndicator target="_blank" />;
   const canadaTermsConditions = <InlineLink to={t('status:links.canada-terms-conditions')} className="external-link" newTabIndicator target="_blank" />;
+  const cite = <cite />;
```

Comment:
A bit of a nitpick, but I don't think this is necessary.. it just adds noise to the source code.